odxtools/diaglayers/diaglayer.py

```diff
@@ -29,9 +29,11 @@
 from ..subcomponent import SubComponent
 from ..unitgroup import UnitGroup
 from .diaglayerraw import DiagLayerRaw
-from .diaglayertype import DiagLayerType
+from .diaglayertype import DiagLayerType, TInheritancePrio
 
 PrefixTree = dict[int, Union[list[DiagService], "PrefixTree"]]
+TInheritedObject = tuple[TNamed, "DiagLayer", TInheritancePrio]
```

Comment:
I think it would be better to create an actual class/named tuple

Reply:
I agree, i will change that.